src/main/java/setup/BaseTest.java

```diff
@@ -1,52 +1,78 @@
 package setup;
 
 import io.appium.java_client.AppiumDriver;
-import org.openqa.selenium.remote.DesiredCapabilities;
-import org.testng.annotations.*;
-import pageObjects.PageObject;
-
 import java.io.File;
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Optional;
+import org.testng.annotations.Parameters;
+import pageobjects.PageObject;
 
 public class BaseTest implements IDriver {
 
     private static AppiumDriver appiumDriver; // singleton
     IPageObject po;
+    WebDriverWait webDriverWait;
+
+    //capability names
+    private static final String PLATFORM_NAME = "platformName";
+    private static final String DEVICE_NAME = "deviceName";
+    private static final String APP = "app";
+    private static final String BROWSER_NAME = "browserName";
+    private static final String CHROME_DISABLE_BUILD_CHECK = "chromedriverDisableBuildCheck";
+
+    //capability values
+    private String chromeDisableBuildCheckBool = "true";
 
     @Override
-    public AppiumDriver getDriver() { return appiumDriver; }
+    public AppiumDriver getDriver() {
+        return appiumDriver;
+    }
 
     public IPageObject getPo() {
         return po;
     }
 
-    @Parameters({"platformName","appType","deviceName","browserName","app"})
+    @Parameters({"platformName", "appType", "deviceName", "browserName", "app"})
     @BeforeSuite(alwaysRun = true)
-    public void setUp(String platformName, String appType, String deviceName, @Optional("") String browserName, @Optional("") String app) throws Exception {
-        System.out.println("Before: app type - "+appType);
+    public void setUp(String platformName, String appType, String deviceName,
+                      @Optional("") String browserName, @Optional("") String app) throws Exception {
+        System.out.println("Before: app type - " + appType);
```

Comment:
System.out.println? Why do you need it here?

Reply:
That was in the original code, I left it just to see what's going on in the console, but now I removed it, thank you